src/components/Sidebar/Sidebar.scss

```diff
@@ -26,4 +26,16 @@
 
 .close {
   transform: translateX(-100%);
-}
\ No newline at end of file
+}
+
+.sidebar-close {
+  display: flex;
+  justify-content: flex-end;
+  font-size: 1.5rem;
```

Comment:
colors, font sizes, generally used paddings, margins, etc, should be placed in the variables.